snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -327,7 +320,7 @@ func (b *bootstrapper) Start(ctx context.Context, startReqID uint32) error {
 		return err
 	}
 
-	b.Config.SharedCfg.RequestID = startReqID
+	b.requestID = startReqID
```

Comment:
Technically that is true _here_... Because this specific `Start` function is always called with `0`... But the snowman bootstrapper does need to honor this to avoid potentially sending multiple messages with the same `nodeID + requestID + op` which could cause a deadlock.